sessions/css-flexbox/personal-travel-diary/css/styles.css

```diff
@@ -0,0 +1,60 @@
+* {
+	box-sizing: border-box;
+}
+
+/* Exercise 1: Adjust the overall layout with Flexbox according to the following criteria:
+
+- The content should be aligned in the center and should stay in the center when changing the size of the viewport.
+- The heading "My Personal Travel Diary" should also be aligned in the center of the page.
```

Comment:
I found this exercise rather confusing as it doesn't seem like the content is actually centered if the articles are not centered. 🤔 Also is it correct and the intended way that there are two flexboxes needed? One on the body and the other one on the main?